weave/trace/op.py

```diff
@@ -140,6 +149,17 @@ async def _run_async() -> Coroutine[Any, Any, Any]:
         else:
             return on_output(res)
 
+    def call(self, *args: Any, **kwargs: Any) -> "Call":
+        client = client_context.weave_client.require_weave_client()
+        run = self._create_run(*args, **kwargs)
```

Comment:
Would be nice to not perpetuate the "run" terminology

Reply:
let's do a second rename PR that changes it to call